README.md

```diff
@@ -34,7 +34,7 @@ number of dependencies for installation. For full details please see [INSTALL.md
 We're always keen to have more developers! Pull requests are very welcome.
 
 * Bugs are recorded in the [issue tracker](https://github.com/openstreetmap/openstreetmap-website/issues).
-* Translation is managed by [Translatewiki](https://translatewiki.net/wiki/Translating:OpenStreetMap).
+* Translation is managed by [Translatewiki](https://translatewiki.net/wiki/Translating:OpenStreetMap), for community names we use data in [OCI](https://github.com/osmlab/osm-community-index) and work on [Transifex](https://www.transifex.com/openstreetmap/id-editor/languages/)
```

Comment:
> Why not make it a separate bullet point and explain what OCI stands for?

Because in general they are all about how the translation of the string is obtained, so I wrote the explanation of the community names follows the previous Translatewiki link. 

I'm not a mapper that pay much attention to this repo, just found this when I translated new deployed osm website. So if you have better suggestions please don't hesitate to show them

Reply:
Well, OCI doesn’t only impact the translation, but is also the place where you’d have to add new communities, local chapters or fix their name, etc. 

That’s why I believe it would make sense to have it as a separate point, rather than adding to another translation item.

Reply:
> Well, OCI doesn’t only impact the translation, but is also the place where you’d have to add new communities, local chapters or fix their name, etc.

Yes, you are right, but maybe there are some misunderstanding, I'm not the person that propose to use OCI data (It was introduced in #3301 ), I'm the person confused by how to find those original string and don't want other translator waste time on finding the correct place, OCI repository's Transifex. So I wrote this PR.

Reply:
Yes, i know. Don’t worry, we’ll address this in another pull request.

Reply:
> Yes, i know. Don’t worry, we’ll address this in another pull request.

So I wander to know whether we should merge this as a temporary description 🤣

Reply:
Well, I would add something like: The list of local chapters on https://www.openstreetmap.org/communities is being maintained on the OSM Community index (OCI). Any changes require a new version of the OCI to be published and the updated component to be deployed as part of openstreetmap-website. Translation of the OCI entries need to be made on Trsnsifex and need to be available at the time of publishing a new OCI version.

Reply:
I've rephrased this to be on a separate list item - as @mmd-osm says, where the list of Chapters come from is of interest to developers, beyond just within the context of i18n.